### PR TITLE
7

### PR DESCRIPTION
**Pull request by schoeyy**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -186,4 +186,53 @@ describe("Articles", () => {
...
+        .expect(201)
+        .then((response) => {
+          const { comment } = response.body;
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            body: expect.any(String),
+            author: expect.any(String),
+          });
```
Comment (Keb84) [inline __tests__/app.test.js:207]: We would expect another test here for a status 201 when a comment is created that has additional properties (for example it's given a username, body, and votes). We would want the extra property to be ignored.

**Review by Keb84 — APPROVED** [review]
Please add the additional tests and resubmit 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -207,6 +207,27 @@ describe("Articles", () => {
...
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          const { comment } = response.body;
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:224]: If you're passing it 4 votes, how do we know the votes property has been ignored? Can you make this test more specific to show the votes property has not increased by 4.

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
Please see small change and resubmit

**Review by Beth-McCormack — APPROVED** [review]

**Pull request merged**